modules/imgcodecs/test/test_exr.impl.hpp

```diff
@@ -23,6 +34,8 @@ TEST(Imgcodecs_EXR, readWrite_32FC1)
     ASSERT_EQ(CV_32FC1,img.type());
 
     ASSERT_TRUE(cv::imwrite(filenameOutput, img));
+    // Check generated file size to ensure that it's compressed with proper options
+    ASSERT_EQ(getFileSize(filenameOutput), 396u);
```

Comment:
`_EQ` parameters order is `(expected, actual)`

Here and below.

Reply:
Done.